scripts/idl/lint/lint_rules_grammar.lark

```diff
@@ -0,0 +1,32 @@
+start: instruction*
+
+instruction: load_xml|all_endpoint_rule|specific_endpoint_rule
+
+load_xml: "load" ESCAPED_STRING ";"
+
+all_endpoint_rule: "all" "endpoints" "{" required_global_attribute* "}"
+
+specific_endpoint_rule: "endpoint" integer "{" required_server_cluster* "}"
+
+required_global_attribute: "require" "global" "attribute" id "=" integer ";"
+
+required_server_cluster: "require" "server" "cluster" id ";"
+
+integer: positive_integer | negative_integer
+
+positive_integer: POSITIVE_INTEGER | HEX_INTEGER
+negative_integer: "-" positive_integer
```

Comment:
I am not familiar with lark, but wouldn't this allow -0x0001?

Reply:
yes, I believe it would. Even knowing this though, maybe it is good enough. I believe used positive_integer because we already have a paraser to convert string -> number. If I use `POSITIVE_INTEGER` instead, then I have to parse here too.